django_project/frontend/views/base_view.py

```diff
@@ -115,17 +116,20 @@ def get_organisation_list(self, request):
                     id=user_profile.current_organisation.id
                 )
         else:
-            user_organisations = OrganisationUser.objects.filter(
-                user=user
-            ).order_by('organisation_id')
-            if user_profile and user_profile.current_organisation:
-                user_organisations = user_organisations.exclude(
-                    id=user_profile.current_organisation.id
+            organisations = []
+            for Model in [OrganisationUser, OrganisationRepresentative]:
```

Comment:
@zamuzakki will this not create duplicate organisations? becacuse manager will have 2records in both table.

Reply:
@danangmassandy Updated to check only from OrganisationUser.